sln/WebApiOData.Performance.Official.sln

```diff
@@ -0,0 +1,38 @@
+﻿
```

Comment:
In Official solution, I remove the "System.Web.OData.Project" reference, because:

1. we download the nuget packages (for example, the System.Web.OData.Project.), we don't need to build it.
2. If we keep the reference in the solution, it will build with errors.
3. I can't find a <choose> element which can be used in solution files.

So, I create a new solution file for the official build.